### PR TITLE
[TS-53] Stop overlapping and responsive styling for event type toggle

### PR DESCRIPTION
**Pull request by William-Unosquare**
- Fixes overlapping of event type toggle & calendar navigation buttons.
- Improves responsiveness of event type toggle.

chrome_2018-09-11_16-36-11

chrome_2018-09-11_16-36-19

chrome_2018-09-11_16-36-28

### CODE REVIEW

**Review by mcromie92 — APPROVED** [review]
looks good

**Comment by ghost** [comment]
@William-Unosquare just looking at the screenshot. Any chance a bit of spacing could be added between the icon and the text 'Team Events'

**Review by ghost — CHANGES_REQUESTED** [review]
Just about the spacing for the icon and Team events text.

File: web-app/src/pages/Dashboard/styled.js [hunk 1]
```diff
@@ -2,11 +2,31 @@ import styled from 'styled-components';
...
   top: 30px;
-  display: flex;
   height: 40px;
-   `;
+  display: none;
+  .btnText {
+    display: none;
+  }
+
+  span {
```
Comment (ghost) [inline web-app/src/pages/Dashboard/styled.js]: any chance we could not use !important?

Comment (ghost) [inline web-app/src/pages/Dashboard/styled.js]: Small thing but try to give everything a class rather than put CSS on a element.

Comment (bgribz) [reply web-app/src/pages/Dashboard/styled.js]: I don't think this is an issue when used with styled-components, maybe even preferable to keep code clean.

Comment (ghost) [reply web-app/src/pages/Dashboard/styled.js]: It's not a stylistic reason, it's just for performance. A small thing one time doesn't make a huge difference but 100 times makes all the difference across an app. Appreciate it's CSS within JS but it should still stand. I've been too quite a few performance talks now and it's always brought up with CSS. No tags and try to stay away from ID's

Comment (ghost) [reply web-app/src/pages/Dashboard/styled.js]: Go with whatever is consistent to what already has been done but it's just good to note for future.

Comment (ghost) [reply web-app/src/pages/Dashboard/styled.js]: @bgribz was beside me and mentioned he thinks a class is generated. I checked https://www.styled-components.com/docs/advanced and inspected a box and it does generate a class. So ignore what I said there in terms of styled components but the principle stands

**Review by ghost — APPROVED** [review]

**Comment by William-Unosquare** [comment]
Cleaned up the CSS. I used class names because it came with the bonus that it solved the !important along with a cascade swap. Also changed btnText to btn-text because it's how the other classes in the file are named.

Also fixed the margin on the icon. Ooops.

**Pull request merged**